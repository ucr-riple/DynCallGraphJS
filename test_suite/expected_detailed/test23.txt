{
    'global (test23.js:10:12:10:37)': [
        'Person (test23.js:1:1:4:2)'
    ],
    'global (test23.js:11:1:11:12)': [
        'anon (test23.js:6:25:8:2)'
    ]
}