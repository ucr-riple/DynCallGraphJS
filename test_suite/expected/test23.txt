{ '(test23.js:10:12:10:37)':
   [ '(test23.js:1:1:4:2)' ],
  '(test23.js:11:1:11:12)':
   [ '(test23.js:6:25:8:2)' ] }